docs/src/examples/expect/example.md

```diff
@@ -0,0 +1,37 @@
+## Examples
+### Use different matchers
+
+This example shows how to use the `expect` function and its different matchers
+
+```solidity
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import {Test, expect} from "vulcan/test.sol";
+
+contract ExpectExample01 is Test {
+    function test() external {
+        expect(string("foo")).toEqual(string("foo"));
```

Comment:
maybe it is better to store the variable, otherwise users might think that you always need to cast it.
```suggestion
        string memory foo = "foo";
        expect(foo).toEqual("foo");
```

Reply:
Also I think the casting is not necessary in the matcher, as matchers are different for each expect(...)